### ISSUE TITLE
Back to top link SVG is focussable in Internet Explorer

### ISSUE THREAD

**Issue by NickColley** [alphagov/govuk-design-system#900]
This issue is from a May 2019 external accessibility audit report.

**WCAG Reference**: Usability feedback only, there is no WCAG related guidelines.
**Issue ID**: DAC_Issue7
**URL**: https://design-system.service.gov.uk/styles/layout/

## Screen Shot
![](https://user-images.githubusercontent.com/2445413/58795191-f117eb80-85f1-11e9-84c7-53b79aaddfae.png)

An ‘svg’ image has been used to present the ‘back to top’ arrow which is currently receiving focus for keyboard only users in Internet Explorer. When selected this took the user to the top of the page. This is a known bug in Internet Explorer, and can be overcome using focusable="false" on the SVG.

## Current Code Ref(s)
```html
<a class="govuk-link govuk-link--no-visited-state" href="#top"> <svg class="app-back-to-top__icon" xmlns="http://www.w3.org/2000/svg" width="13" height="17" viewBox="0 0 13 17"> <path fill="currentColor" d="M6.5 0L0 6.5 1.4 8l4-4v12.7h2V4l4.3 4L13 6.4z"></path> </svg>Back to top </a>
```

## Screen reader comments
> “I found that, when in Internet Explorer using JAWS, I tabbed through the page with the ‘Tab’ key and the ‘Back to top’ and ‘gov.uk design system’ links appeared twice within the tab order despite only appearing once in context.”

## Keyboard only comments
> “There is a missing tab between the ‘back to top’ and the ‘open the example in a new window’ link. At this point a keyboard only user does not know where their focus has gone. This is also the case between the ‘open this example in a new window’ and the ‘back’ link, and happens throughout the page.”

## Solution
Add focusable="false" to the svg to ensure that it does not receive focus for users in Internet

**Comment by nickcolley** [comment]
Easy to do, let's sort it as soon as...

**Comment by nickcolley** [comment]
Confirmed as fixed in a re-test by DAC on 7th October 2019
